RobotInterrogation/ClientApp/src/components/interviewParts/OutcomeHumanCorrect.tsx

```diff
@@ -0,0 +1,40 @@
+import * as React from 'react';
+import { InterviewPosition } from '../interviewReducer';
+import { Typography } from '@material-ui/core';
+import { Page } from './elements/Page';
+import { P } from './elements/P';
+
+interface IProps {
+    position: InterviewPosition;
+}
+
+export const OutcomeHumanCorrect: React.FunctionComponent<IProps> = (props) => {
+    switch (props.position) {
+        case InterviewPosition.Interviewer:
+            return (
+                <Page>
+                    <P>You correctly identified the suspect as a human.</P>
+                    <Typography variant="h4">You both win.</Typography>
+                </Page>
+            );
+
+        case InterviewPosition.Suspect:
+            return (
+                <Page>
+                    <P>The interviewer correctly identified you as a human.</P>
+                    <Typography variant="h4">You both win.</Typography>
+                </Page>
+            );
+
+        case InterviewPosition.Spectator:
+            return (
+                <Page>
+                    <P>The interviewer correctly identified the suspect as a human.</P>
+                    <Typography variant="h4">They both win.</Typography>
+                </Page>
+            );
+
+        default:
+            return <P>Unknown outcome</P>;
```

Comment:
I'd be tempted to combine the `default` case with the `Spectator` case here.

If you're somehow in InterviewPosition.None, there's no harm in showing the spectator result, IMO.